content/events/contents.lr

```diff
@@ -4,8 +4,9 @@ body:
 
 # Events
 
-## 2020
+## 2020 (CURRENT)
```

Comment:
```suggestion
## 2020
```
no need to denote current, we don't need to update this in 2021 if we don't note it as current